internal/template-validator/validator/app.go

```diff
@@ -130,7 +130,10 @@ func (app *App) Run() {
 
 func registerReadinessProbe() {
 	http.HandleFunc("/readyz", func(resp http.ResponseWriter, req *http.Request) {
-		fmt.Fprintf(resp, "ok")
+		_, err := fmt.Fprintf(resp, "ok")
```

Comment:
`resp.Write([]byte("ok"))` would probably achieve the same without an additional dependency.

Reply:
Good idea, done.